nerfstudio/viewer/server/viewer_utils.py

```diff
@@ -325,7 +325,7 @@ def init_scene(self, dataset: InputDataset, start_train=True) -> None:
         # set the initial state whether to train or not
         self.vis["renderingState/isTraining"].write(start_train)
 
-        self.vis["renderingState/render_time"].write(str(0))
+        # self.vis["renderingState/render_time"].write(str(0))
```

Comment:
Remove? Or add todo

Reply:
There are other comments below in this file. I think it's fine to leave in for this particular case.